documentation/examples/column-format/no_matchers.ts

```diff
@@ -0,0 +1,16 @@
+import * as coda from '../../../index';
+
+const pack = coda.newPack();
+
+// BEGIN
+
+// Replace all <text> with your own text
+// a custom column type that you apply to any column in any Coda table.
+pack.addColumnFormat({
+  name: '<format name>',
+  formulaNamespace: '<formulaNamespace of Pack>', // this will be deprecated
```

Comment:
Ah yeah, we should (separately) probably change `addColumnFormat()` so that it doesn't require a namespace, and populates it with a dummy default one if you don't provide one, so that users don't have to think about this during the deprecation period.